setup.cfg

```diff
@@ -15,30 +15,30 @@ keywords =
     pipenv
     pip-compat
 classifier =
-    Development Status :: 4 - Beta
+    Development Status :: 5 - Production
     License :: OSI Approved :: ISC License (ISCL)
     Operating System :: OS Independent
     Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.6
     Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.4
     Programming Language :: Python :: 3.5
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
     Topic :: Software Development :: Build Tools
     Topic :: Software Development :: Libraries :: Python Modules
     Topic :: Utilities
 
 [options]
 zip_safe = true
-python_requires = >=2.6,!=3.0,!=3.1,!=3.2,!=3.3
+python_requires = >=2.6,!=3.0,!=3.1,!=3.2,!=3.3,!=3.4
```

Comment:
```suggestion
python_requires = >=2.7,!=3.0,!=3.1,!=3.2,!=3.3,!=3.4
```

Reply:
er yeah i caught that one but opening a new pr